memcached-spring-boot-autoconfigure/build.gradle

```diff
@@ -15,7 +15,9 @@ dependencies {
     optionalImplementation 'org.springframework.boot:spring-boot-actuator'
     optionalImplementation 'org.springframework.cloud:spring-cloud-context'
     optionalImplementation 'io.micrometer:micrometer-core'
+    implementation 'org.springframework:spring-context-support'
 
+    testImplementation 'javax.cache:cache-api:1.1.1'
```

Comment:
You might not need to specify the version i.e. it should be resolved from the Spring Boot Dependency pom file

Reply:
Good catch!